test/classes/initializers/constAndParamInHeader.good

```diff
@@ -0,0 +1,4 @@
+constAndParamInHeader.chpl:6: In initializer 'Foo':
+constAndParamInHeader.chpl:6: error: invalid access of class member in initializer header
+constAndParamInHeader.chpl:14: In initializer 'Bar':
+constAndParamInHeader.chpl:14: error: invalid access of class member in initializer header
```

Comment:
That's the wording from the original error message.  If I recall correctly, I just modified the original error message to say "initializer" instead of "constructor"